
Defaulting to extended NSVF dataset
LOAD NSVF DATA /hpc/users/CONNECT/haotianbai/work_dir/AdaptiveNerf/data/nerf_synthetic split train
Traceback (most recent call last):
  File "opt.py", line 269, in <module>
    dset = datasets[args.dataset_type](
  File "/hpc/users/CONNECT/haotianbai/work_dir/AdaptiveNerf/svox2/opt/util/dataset.py", line 20, in auto_dataset
    return NSVFDataset(root, *args, **kwargs)
  File "/hpc/users/CONNECT/haotianbai/work_dir/AdaptiveNerf/svox2/opt/util/nsvf_dataset.py", line 82, in __init__
    img_dir_name = look_for_dir(["images", "image", "rgb"])
  File "/hpc/users/CONNECT/haotianbai/work_dir/AdaptiveNerf/svox2/opt/util/nsvf_dataset.py", line 79, in look_for_dir
    assert False, "None of " + str(cands) + " found in data directory"
AssertionError: None of ['images', 'image', 'rgb'] found in data directory
